src/utils/is-stack-focused.util.ts

```diff
@@ -0,0 +1,5 @@
+import { ScreensOrModalsEnum } from '../interfaces/stacks.interface';
+import { isDefined } from './is-defined';
+
+export const isStackFocused = (currentRouteName: ScreensOrModalsEnum, screensStack: ScreensOrModalsEnum[]) =>
```

Comment:
could we use `useIsFocused` from react-navigation? 
https://reactnavigation.org/docs/use-is-focused/